index.html

```diff
@@ -3138,9 +3138,95 @@ <h2>DID Resolution</h2>
 consumption rules.">
         <figcaption>
 Functions resolve() and resolveRepresentation().
+See also: <a class="longdesc-link"
+href="#resolve-resolverepresentation-longdesc">narrative description</a>.
         </figcaption>
       </figure>
 
+        <div class="longdesc" id="resolve-resolverepresentation-longdesc">
+            <p>
+                The upper middle part of the diagram contains a rectangle with dashed grey outline, containing two
+                blue-outlined rectangles, one above the other, as follows.
```

Comment:
```suggestion
                blue-outlined rectangles, one above the other.
```

Reply:
I copied this part of the text from @clehner 's PR https://github.com/w3c/did-core/pull/781/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R2695. Would you also suggest the same change there?

Reply:
Yes.  Done.